src/resonanceReconstruction/rmatrix/CompoundSystem.hpp

```diff
@@ -47,6 +49,11 @@ class CompoundSystem {
 
   auto spinGroups() const { return ranges::view::all( this->groups_ ); }
 
+  /**
+   *  @brief Return the reactions defined in the compound system
+   */
+  auto reactionIDs() const { return ranges::view::all( this->reactions_ ); }
```

Comment:
Why use `ranges::view::all` here instead of just returning `reactions_`?

Reply:
I'm trying to keep consistency in the coding practices and style. We've been using ranges in resonance reconstruction consistently and I did not want to move away from that just now (at least not without changing everything at the same time).